Corrections/Correction_C#.cs

```diff
@@ -157,39 +168,52 @@ static void Main(string[] args)
             int[][] multiArray = new int[3][];
 
             for (int i = 0; i < multiArray.Length; i++)
+            {
                 multiArray[i] = new int[3];
+            }
 
             for (int i = 0; i < multiArray.Length; i++)
+            {
                 for (int j = 0; j < multiArray[i].Length; j++)
-                    multiArray[i][j] = rdmBounds(0, 10);
-            
-            
-            for (int i = 0; i  <multiArray.Length; i++)
+                {
+                    multiArray[i][j] = rnd(0, 10);
+                }
+            }
+
+            for (int i = 0; i < multiArray.Length; i++)
             {
                 Console.Write("{");
 
                 for (int j = 0; j < multiArray[i].Length; j++)
+                {
                     Console.Write(multiArray[i][j] + ",");
-                    
+                }
+
                 Console.WriteLine("}");
             }
-            
-            
+
             Console.Read(); //force the Console to wait before closing
         }
 
         static void TriForce()
         {
-            Console.WriteLine("   /\\  ");
-            Console.WriteLine("  /__\\ ");
-            Console.WriteLine(" /\\  /\\  ");
-            Console.WriteLine("/__\\/__\\ ");
-            Console.WriteLine("TRIFORCE");
+            Console.WriteLine(@"
+   /\
+  /__\ 
+ /\  /\ 
+/__\/__\  
+TRIFORCE"
+);
```

Comment:
In fact, till I use a verbatim string here instead of multiple Console.Writeline, it's not an indentation issue but just the specs application ;)